Added support gating to historic data import pages (#8030)
PostHog/posthog.com: contents/docs/migrate/ingest-historic-data.mdx

showTitle: true
---

> > Prior to beginning a historical data import please ensure that you've done the following:

Reviewer: ```suggestion
> Prior to beginning a historical data import please ensure you do the following:
```

> 1. Create a project on our [US](https://us.posthog.com/signup) or [EU](https://eu.posthog.com/signup) Cloud

> > 2. Sign up to a paid plan on the billing page in-app (historic imports are not billable but this will unlock the necessary features).

Reviewer: ```suggestion
> 2. Sign [up to a paid plan on the billing page](https://us.posthog.com/organization/billing) (historic imports are not billable but this will unlock the necessary features).
```

Author: @joethreepwood this could be confusing if they have cloud accounts on US and EU, I see what you're doing as if there's only one account we should redirect but if there are two they could end up doing it in the wrong place.
